quickstarts/mean/config.yml

```diff
@@ -0,0 +1,52 @@
+
+slug: mean-stack
+description: |
+  ## Comprehensive MEAN stack monitoring system
+
+  MEAN stack is an open source JavaScript-based web application development framework for building dynamic websites and web applications.
```

Comment:
Perhaps MEAN should be defined here? 
e.g something like:

"MEAN stack is an open source JavaScript-based web application development framework for building dynamic websites and web applications.  The stack consists of MongoDB, ExpressJS, AngularJS and NodeJS."